client/src/app/components/timestamp/timestamp.tsx

```diff
@@ -0,0 +1,21 @@
+import { formatTimestamp } from '../../utils/formatters';
+import { HistoryLink } from '../history-link/history-link';
+import { BodyText } from '../body-text/body-text';
+
+interface TimestampProps {
+	deployHash?: string;
+	timestamp: string;
+}
+
+const Timestamp = ({ deployHash, timestamp }: TimestampProps) => {
```

Comment:
It's better to rename this component to `PlayTimestamp` and pass the whole `play` as the argument. It's not like we will render any generic timestamp like this.

Reply:
@victoriari then maybe let's rename it to `DeployTimestamp`?